keyboards/latinpad/readme.md

```diff
@@ -0,0 +1,10 @@
+macro 2 rotary encoder pad
+
+Keyboard Maintainer: Latincompass
+Hardware Availability: https://github.com/latincompass/qmk_firmware/tree/master/keyboards/latinpad
+
+Make example for this keyboard (after setting up your build environment):
+
+make latinpad:default
+
+See the build environment setup and the make instructions for more information. Brand new to QMK? Start with our Complete Newbs Guide.
```

Comment:
```suggestion
# Latinpad

macro 2 rotary encoder pad

* Keyboard Maintainer: Latincompass
* Hardware Availability: https://github.com/latincompass/qmk_firmware/tree/master/keyboards/latinpad

Make example for this keyboard (after setting up your build environment):

    make latinpad:default

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).

```

Reply:
URL should be a link to their hardware, not their fork of the QMK firmware.